### PR TITLE
Use CentOS packages on RHEL

### PR DESCRIPTION
**Pull request by zmc**
http://tracker.ceph.com/issues/11772

http://qa-proxy.ceph.com/teuthology/zack-2015-05-26_15:15:15-teuthology:ceph-master---basic-vps/910011/teuthology.log

### CODE REVIEW

**Comment by zmc** [comment]
As described in the ticket, this depends on having CentOS 7 packages built with '.el7' instead of '.el7.centos' to work properly.

**Comment by zmc** [comment]
test this please

**Comment by zmc** [comment]
Looks to be working:

http://qa-proxy.ceph.com/teuthology/zack-2015-05-26_15:17:27-teuthology:ceph-master---basic-multi/910013/teuthology.log

File: teuthology/task/install.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def _get_baseurlinfo_and_dist(ctx, remote, config):
             'lsb_release', '-rs'], stdout=StringIO())
     retval['relval'] = r.stdout.getvalue().strip()
     dist_name = None
-    if retval['distro'] == 'CentOS':
+    if retval['distro'] in ('CentOS', 'RedHatEnterpriseServer'):
         relval = retval['relval']
         relval = relval[0:relval.find('.')]
         distri = 'centos'
```
Comment (liewegas) [inline teuthology/task/install.py]: should this be 'el' now?

Comment (zmc) [inline teuthology/task/install.py]: No, that's ``dist_name``

**Pull request merged**